### ISSUE TITLE
question about run “ucf101_testing.sh ”

### ISSUE THREAD

**Issue by sophiazy** [facebook/C3D#215]
when I run the create_volume_mean.sh ,the following question is:

I0313 09:25:57.691403 20799 net.cpp:112] accuracy -> accuracy
I0313 09:25:57.691413 20799 net.cpp:127] Top shape: 1 2 1 1 1 (2)
I0313 09:25:57.691421 20799 net.cpp:154] accuracy needs backward computation.
I0313 09:25:57.691428 20799 net.cpp:165] This network produces output accuracy
I0313 09:25:57.691457 20799 net.cpp:183] Collecting Learning Rate and Weight Decay.
I0313 09:25:57.691478 20799 net.cpp:176] Network initialization done.
I0313 09:25:57.691486 20799 net.cpp:177] Memory required for Data 5653135008
E0313 09:25:59.527685 20799 test_net.cpp:45] Running 838 iterations.
E0313 09:26:02.303064 20799 test_net.cpp:51] Batch 0, accuracy: 0.76
E0313 09:26:04.814565 20799 test_net.cpp:51] Batch 1, accuracy: 0.74
E0313 09:26:07.333701 20799 test_net.cpp:51] Batch 2, accuracy: 0.78
E0313 09:26:09.845320 20799 test_net.cpp:51] Batch 3, accuracy: 0.76
E0313 09:26:12.370980 20799 test_net.cpp:51] Batch 4, accuracy: 0.72
E0313 09:26:14.882516 20799 test_net.cpp:51] Batch 5, accuracy: 0.8
F0313 09:26:15.006660 20850 video_data_layer.cpp:123] Check failed: read_status Testing must not miss any example
*** Check failure stack trace: ***
Aborted (core dumped)

I would appreciated for some advice,very thanks!

**Comment by dutran** [comment]
We have an assert (at video_data_layer.cpp:123) to make sure we don't miss (read fail) any example in testing. This is mainly for feature extraction. If you test set is large enough, you can comment this line out, the accuracy will not much different, but if your test set is small, missing some examples, will significantly change your final results.

Again, in feature extraction, missing one example may cause the feature mismatch.

**Comment by sophiazy** [comment]
my test set is the original "test_01.lst" , I only change the path.as for the feature extraction ,I have not extract feature ,I just fintuning on the ucf101 with the initialization of "conv3d_deepnetA_sport1m_iter_1900000"
